packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react'
+import { path, prop, propEq } from 'ramda'
+import { FormattedMessage } from 'react-intl'
+import { Text } from 'blockchain-info-components'
+
+export const getTiming = (state, tier) => {
```

Comment:
does not belong to model but to some component messaging service

Reply:
I will move to services.js file